administrator/components/com_associations/models/forms/filter_associations.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<field
+		name="associationcomponent"
+		type="associatedcomponent"
+		label="COM_ASSOCIATIONS_COMPONENT_SELECTOR_LABEL"
+		description="COM_ASSOCIATIONS_COMPONENT_SELECTOR_DESC"
+		onchange="this.form.submit();"
+		>
+		<option value="">COM_ASSOCIATIONS_FILTER_SELECT_ITEM_TYPE</option>
+	</field>
+	<field
+		name="associationlanguage"
+		type="contentlanguage"
+		label="JOPTION_FILTER_LANGUAGE"
+		description="JOPTION_FILTER_LANGUAGE_DESC"
+		onchange="this.form.submit();"
+		>
+		<option value="">JOPTION_SELECT_LANGUAGE</option>
+	</field>
+	<fields name="filter">
+		<field
+			name="search"
+			type="text"
+			label="COM_ASSOCIATIONS_FILTER_SEARCH_LABEL"
+			description="COM_ASSOCIATIONS_FILTER_SEARCH_DESC"
+			hint="JSEARCH_FILTER"
+		/>
+	</fields>
+
+	<fields name="list">
+		<field
+			name="fullordering"
+			type="list"
+			label="COM_ASSOCIATIONS_LIST_FULL_ORDERING"
+			description="COM_ASSOCIATIONS_LIST_FULL_ORDERING_DESC"
```

Comment:
just noticed this language vars doesn't exist in joomla.
so please remove label and description attributes